app/views/admin/edition_change_notes/edit.html.erb

```diff
@@ -0,0 +1,35 @@
+<% content_for :page_title, "#{@edition.title}: Change notes" %>
+<% content_for :context, @edition.title %>
+<% content_for :title, "Change notes" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">Change published at: <%= I18n.l(@edition_to_change.major_change_published_at, format: :long_ordinal) %></p>
+    <p class="govuk-body">Current change note: <%= @edition_to_change.change_note %></p>
+
+    <%= form_for @edition_to_change, method: :patch, url: admin_edition_change_note_path(@edition_to_change) do |form| %>
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "New change note:"
+        },
+        name: "new_change_note",
+        value: @edition_to_change.change_note,
+      } %>
+
+      <div class="govuk-button-group govuk-!-margin-bottom-6">
```

Comment:
Should probably use "Save" and "Cancel" as per other pages.
https://github.com/alphagov/whitehall/blob/main/app/views/admin/attachments/_form.html.erb#L78-L90

Reply:
Thanks, I'll update the label and add a 'Cancel' link.